src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -238,6 +238,7 @@ function DesktopTriggers() {
           return (
             <TriggerButton
               ariaLabel={item.ariaLabels?.triggerButton}
+              tooltipText={item.ariaLabels?.drawerName}
```

Comment:
Note to self, a new test should assert the Aria Label works as the default if no tooltipText given